docs/networks/arbitrum-goerli.md

```diff
@@ -47,7 +47,7 @@ testnet (for now) are Mainnet subgraphs. This means:
 | `INDEXER_AGENT_INDEXER_ADDRESS`             | `--indexer-address`             | Ethereum address of testnet indexer                                                         |
 | `INDEXER_AGENT_INDEXER_GEO_COORDINATES`     | `--indexer-geo-coordinates`     | Geo coordinates of testnet indexer infrastructure                                           |
 | `INDEXER_AGENT_MNEMONIC`                    | `--mnemonic`                    | Ethereum mnemonic for testnet operator                                                      |
-| `INDEXER_AGENT_NETWORK_SUBGRAPH_DEPLOYMENT` | `--network-subgraph-deployment` | `QmTWLwhFsDdi4MuhDQzNuTsGC6WBRKk9NeQL6nRz8yTari`                                            |
+| `INDEXER_AGENT_NETWORK_SUBGRAPH_DEPLOYMENT` | `--network-subgraph-deployment` | `QmexmwCR62YRBzWXNaT3Lr69L6QYrWz5mrouhxgWfvxDQX`                                            |
```

Comment:
did you mean to do this in the Arbitrum One config? (this one's arb goerli)

Reply:
fixed